### DISCUSSION TITLE
tab_plot Alternative

### DISCUSSION THREAD

**Discussion by abdurrafiahsan** [wannier-berri/wannier-berri#347] [Q&A]
Dear wannierberri developer and community,

I tried to plot the Berry Curvature along a certain K-path using "Plotting lines and planes (tab_plot)" utility, but it seems that the tab_plot no longer exist, is there any alternative to plot the pickle data?

Best regards,

Abdurrafi Ahsan

**Comment by stepan-tsirkin** [comment]
Sure, see this tutorial

https://tutorial.wannier-berri.org/tutorials/1_basic/tutorial-wb-basic-solution.html#Interpolation-on-a-path
